script/create_person.py

```diff
@@ -0,0 +1,101 @@
+#!/usr/bin/env python
+"""
+This script attempts to create a legacy user with SSHA password on a cnx
+postgres db
+
+1. You need to download this script onto the server, e.g. staging08.cnx.org,
```

Comment:
Erm... I'm not sure about running this on staging so we probably shouldn't be using `staging08.cnx.org` as an example.

There's already a way to create a user in legacy which should create an entry in the `persons` table so this script should be purely for development not on servers.

https://github.com/openstax/devops/wiki/Adding-Accounts-to-Legacy-Staging

Reply:
:+1:

Reply:
I've made the change in the comment